api/v1/apiserver_types.go

```diff
@@ -173,6 +180,12 @@ type APIServerDeployment struct {
 
 // APIServerDeploymentSpec defines configuration for the API server Deployment.
 type APIServerDeploymentSpec struct {
+	// Replicas defines how many instances of the API server pod will run.
+	// +optional
+	// +kubebuilder:validation:Minimum=0
+	// +kubebuilder:validation:Maximum=2147483647
+	Replicas *int32 `json:"replicas,omitempty"`
```

Comment:
The replicas for the apiserver can already be controlled through the Installation resource with the `controlPlaneReplicas` field.
I'm not saying this wouldn't be useful but just wanted to point out it is possible to change the number of replicas.

Reply:
you are right! Removing these lines, thanks for pointing this out.